src/Makefile.am

```diff
@@ -90,8 +96,14 @@ test_page_list_SOURCES = \
     s3fs_logger.cpp \
     string_util.cpp \
     test_page_list.cpp
+if MSYS
+    test_page_list_SOURCES += strptime.cpp
+endif
 
 test_string_util_SOURCES = string_util.cpp test_string_util.cpp s3fs_logger.cpp
+if MSYS
+    test_string_util_SOURCES += strptime.cpp
```

Comment:
Does it make more sense to add the `strptime.cpp` polyfill to `string_util.cpp`?  This will reduce the amount of Windows conditionals.

Reply:
Moved the polyfill into `string_util.cpp` at d8092b4.